bin/deb_local

```diff
@@ -252,59 +257,61 @@ if o_deregister
     exit 0
 end
 
-Autoproj.silent = true
-root_dir  = Autoproj::CmdLine.initialize_root_directory
-selection = Autoproj::CmdLine.initialize_and_load(nil)
+autoprojadaptor.init_autoproj
```

Comment:
that should either be part of the initialize in the pkg_info_ask or rename function to init_database